subsys/fs/nffs_fs.c

```diff
@@ -219,7 +219,9 @@ static int inode_to_dirent(struct nffs_inode_entry *inode,
 		entry->size = 0;
 	} else {
 		entry->type = FS_DIR_ENTRY_FILE;
-		nffs_inode_data_len(inode, &size);
+		rc = nffs_inode_data_len(inode, &size);
+		if (rc)
```

Comment:
missing {}

Reply:
ok, i see this is being followed in this file

But as per linux coding guidelines : 
https://www.kernel.org/doc/html/v4.15/process/coding-style.html

> Do not unnecessarily use braces where a single statement will do.
> 
> if (condition)
>         action();

Also https://google.github.io/styleguide/cppguide.html

>  In general, curly braces are not required for single-line statements

Also skipping braces in single stmt reduces loc, although braces might  make it less error prone.
So is it mandatory that "if" must always have an accompanying brace in zephyr?

Reply:
https://docs.zephyrproject.org/latest/contribute/index.html#coding-style

>Add braces to every if and else body, even for single-line code blocks. Use the --ignore BRACES flag to make checkpatch stop complaining.